projects/core/src/occ/occ-models/occ.models.ts

```diff
@@ -4276,4 +4276,31 @@ export namespace Occ {
     pagination?: PaginationModel;
     sorts?: SortModel[];
   }
+
+  export interface OrgUnitCustomer {
```

Comment:
we can use already created B2BUser with missed props
    orgUnit?: B2BUnit;
    roles?: string[];
    selected?: boolean;